roles/de-condor-launcher/tasks/update.yml

```diff
@@ -6,7 +6,7 @@
   shell: docker pull discoenv/condor-launcher:{{docker.tag}}
 
 - name: pull configs image
-  shell: docker pull gims.iplantcollaborative.org:5000/de-configs-{{environment_name}}:{{docker.tag}}
+  shell: docker pull {{ docker.registry.base }}/de-configs-{{environment_name}}:{{docker.tag}}
```

Comment:
I like this. One thing I haven't been sure of in the migration process is how to differentiate the underlying registry and the proxy; in this case either one would probably work (at least, if the host has `docker login`'d for the auth proxy), but as I'd mentioned the authenticating proxy deliberately blocks pushes.

In any case, I like this!